test/middleware/spec.back-link.js

```diff
@@ -168,6 +175,7 @@ describe('Back Links', function () {
 
     it('returns undefined if referrer header is not on whitelist', function () {
         req.get.withArgs('referrer').returns('http://example.com/not-whitelisted');
+        req.baseUrl = '/not-whitelisted';
```

Comment:
This test is specifically about the referrer header. Adding in baseUrl confuses the purpose of test.